src/modules/peek/Peek.CLI/Peek.CLI.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Peek.CLI", "Peek.CLI.csproj", "{B4A0030D-91C6-4A6D-9CF6-D10F9F7E92D3}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
```

Comment:
drop x86

Reply:
I'll remove the whole sln. That was committed by accident.